tests/e2e-complex/upgrade/common.sh

```diff
@@ -159,6 +161,38 @@ function send_did_new () {
     echo "$txhash" >> $FNAME_TXHASHES
 }
 
+# Send resource
+# input: resource to write
+function send_resource_new () {
+    collection_id_to_write=$1
+    resource_to_write=$2
+
+    # Generate Alice identity key
+    ALICE_VER_KEY="$(cheqd_noded_docker debug ed25519 random)"
+    ALICE_VER_PRIV_BASE_64=$(echo "${ALICE_VER_KEY}" | jq -r ".priv_key_base_64")
+
+    # Build CreateDid message
+    KEY_ID="${collection_id_to_write}#key1"
+
+    RESOURCE_NAME="Resource 1"
+    RESOURCE_RESOURCE_TYPE="CL-Schema"
+    RESOURCE_DATA='{ "content": "test data" }';
+
+    # Post the message
+    # shellcheck disable=SC2086
+    resource=$(local_client_tx tx resource create-resource \
+    --collection-id ${collection_id_to_write} \
+    --resource-id ${resource_to_write} \
+    --resource-name "${RESOURCE_NAME}" \
+    --resource-type "${RESOURCE_RESOURCE_TYPE}" \
+    --resource-file (echo "${RESOURCE_DATA}") \
```

Comment:
Can you echo inline like this or should it be path to a file?

Reply:
No, `resource-file` flag can use only files